Easy/TwoIntegerSum/solution.js

```diff
@@ -0,0 +1,44 @@
+class Solution {
+    /**
+     * Solução
+     *  - Primeiro passo: Criei um hashmap para armazenar informações de cada elemento do array passado
+     *      valor = complemento(um valor que somado com o do elemento resulta no target).
+     *      chave = posição do elemento dentro do array nums.
+     * 
+     *  - Segundo passo: Criei um loop para pega um elemento do array nums, calcula seu complemento e adicionar
+     *      o valor e chave dentro do hashmap.
+     * 
+     *  - Terceiro passo: Fiz uma verificação dentro do loop, se seu complemento se encontra dentro do hashmap é retornado o elemento e
+     *      seu complemento.No entanto, caso seja encontrado o complemento é retornado um array vazio.
+     * 
+     *  Big O Espaço: O(n)
+     *  Big O Tempo: O(n)
```

Comment:
Agora que estamos entendendo um pouco melhor o `Big O`, acho que podemos começar a colocar a descrição do motivo deles, por exemplo:

```suggestion
     *  Big O Espaço: O(n) - Pior cenário eu ter que amazenar todos menos um de numeros em um hashmap 
     *  Big O Tempo: O(n) - Pior cenário se os dois últimos elemento da lista for o que eu procuro
```